frontend/src/components/MeetingRoom/ReportIssue/FeedbackForm/HelperText/HelperText.tsx

```diff
@@ -40,7 +40,7 @@ const HelperText = ({
   };
 
   return (
-    <Box display="flex" justifyContent="space-between" width="100%">
+    <Box component="span" display="flex" justifyContent="space-between" width="100%">
```

Comment:
Was this a span before? I think technically spans aren't meant to contain child elements

Reply:
this was just a `<p>` before with some `<div>`s inside of it and it was failing with: 
```
In HTML, <div> cannot be a descendant of <p>.
This will cause a hydration error.
```

Reply:
This should be okay. The component will be a span containing more spans.

Reply:
The semantics are odd.  `Box` implies "block level" element.  By contrast, `span` implies "inlined" elements, like Mike mentioned.

Do we need to add `component="span"`?

Reply:
yes I think we need to add that `component="span"` here since this `HelperText` component is used inside of another component, `FeedbackForm.` the `FeedbackForm` consists out of a bunch of `<p>`s and nesting a `<div>` inside of it causes a hydration error (see comment above). That's why we need to set `Box` components to `span` so rendering looks like this:
```
<p>
    <span>...</span>
    <span>...</span>
</p>
<p>
    ....
</p>
```